### PR TITLE
Fix clippy warnings

### PR DESCRIPTION
**Pull request by Luni-4**

### CODE REVIEW

**Comment by skydancer0919** [comment]
hello,
i want just inputmapper files. all of them will you send my mail address?

Luni-4 <notifications@github.com>, 8 Nis 2020 Çar, 12:18 tarihinde şunu
yazdı:

File: src/web/server.rs [hunk 1]
```diff
@@ -588,7 +588,9 @@ mod tests {
     #[actix_rt::test]
     async fn test_web_comment_plain_bad_chars() {
         let bad_bytes = &[142, 137, 138, 136, 140, 141, 10];
+        #[allow(clippy::string_lit_as_bytes)]
```
Comment (calixteman) [inline src/web/server.rs:591]: nit: b"..." should be ok + adding bad_bytes: &[u8]

File: src/tools.rs [hunk 2]
```diff
@@ -68,10 +69,10 @@ fn mode_to_str(mode: &[u8]) -> Option<String> {
     std::str::from_utf8(mode)
         .ok()
         .map(|m| m.to_lowercase())
-        .map(|m| m.to_string())
+        .map(|m| m)
```
Comment (calixteman) [inline src/tools.rs:72]: the lambda is identity so could be removed

File: src/web/server.rs (hunk 1 above)
Comment (Luni-4) [reply src/web/server.rs:591]: I can `concat` the two arrays because they have different size. How should I address that?

Comment (calixteman) [reply src/web/server.rs:591]: I just tryed:
```rust
let bad_bytes: &[u8] = &[142, 137, 138, 136, 140, 141, 10];
let input_vec = [b"/*char*/s: ", bad_bytes].concat();
```
and it's ok

Comment (Luni-4) [reply src/web/server.rs:591]: Oh, I forgot the & :sweat_smile:

**Comment by Luni-4** [comment]
@calixteman 

Can we merge this one?

**Comment by Luni-4** [comment]
Rebased!

**Pull request merged**